app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -21,6 +21,7 @@ def schedule_job_multi_button(*args)
     end
 
     def rex_host_features(*args)
+      return unless authorized_for(permission: 'create_job_invocations', auth_object: args.first)
```

Comment:
Should the same also be done in `schedule_job_button` below?

Reply:
Good idea :+1: Done that.